### DISCUSSION TITLE
PSYCHOSIS EFFECT MEGATHREAD

### DISCUSSION THREAD

**Discussion by Udrakan** [FakeFishGames/Barotrauma#4884] [Ideas and suggestions]
Post and discususs your suggestions and feedback for psychosis effects here. It's possible to highlight some highly agreed upon/cool/currently bugged effects here on top.

**Comment by Udrakan** [comment]
**Small leak in the wall**
-minstrength 15
Description:
There is a fake small leak for the player, which causes him to hear water coming in from that spot and it looks like slightly damaged outer shell. Because it's not a catastrophic leak to which more people would react, it's a more believable effect than room being filled with water. Also, it might lead to some nice interactions with crew, when people see patient welding walls with full HP or the player asking others to help him fix a non-existent leak.

**Comment by RadCatish** [comment]
I like the idea of psychosis, but think it should be made more dangerous (probably by making some parts less extreme as suggested above). Maybe I'm just a terrible doctor, but seeing as psychosis doesn't actually effect anything and goes away on its own, I never treat it, rather tell the person to go sit down for a bit. If the effects were more subtle, or more likely to make the psychotic player damage people or use up resources, I think it would be more worthwhile. Some ideas for this:
- Psychotic player sees other players pull a gun on them (making them likely to shoot)
- Welding tool and plasma cutters switch icons in psychotic player's inventory, so if they see a fake leak they're likely to actually damage the hull while trying to fix it
- Psychotic player auto-reports issues they see in rooms, so making their psychosis more annoying for the rest of the crew
- Psychotic players see healthy plants as dead
- Psychotic players see full consumables (fuel rods, oxygen) as empty and vice versa

Basically if psychosis was more likely to make a player inadvertently damage the ship, and was more subtle so it wasn't instantly ruled as psychosis, I think it would be much more interesting! I'd also finally want to craft Haloperidol, and its always good when game elements get utilised.

**Reply by Udrakan** [reply]
Seeing empty consumables as full or not being able to distinguish between welding fuel and O2 tanks might lead to a lot of fun situations.

**Reply by Veritius** [reply]
Auto-report wouldn't be challenging, just irritating. And people might just shoot the psychotic person to get them to shut up.

**Comment by NilanthAnimosus** [comment]
I still find psychosis too easy to distinguish from reality. water in rooms where you see no discernable source for example.

Honestly, I feel it should almost be its own simulated layer such as creating leaks that spread between rooms. flowing water essentially with sounds and all. I know its unlikely for such psychosis to be implemented, but if I see a small fire that doesn't have any obvious source and water that does not flow nor has a visible source (thus unrealistic water) I just instantly snap-realize its all fake and move along.

It is indeed the least believable effect, which if your going insane - your meant to believe it yourself or get tricked on some level, and look like a rambling lunatic to others. realistic-fake water simulation would probably be a start. as well as perhaps welding fake damage to seal fake gaps.

Another thought I would have is fake broken items that you can 'repair' but instead it actually causes a degree of damage to the item (as its not damaged, IE. 'fixing' a 100% health junction takes it down a random value of condition, potentially even breaking it and for some items causing fires or electric shock) - This would make psychosis pretty dangerous as bots and players alike may fudge things over and trust stuff less likely, causing a somewhat incapable crew if widespread.

I would link the "Fake fire" to the "Fake damaged systems" so one also causes the other, as well as keeping the fires in believable locations - I would even have the fake fire cause other fake problems with the submarine such as fake damaging other objects.

If wanting fire in unbelievable locations, I would have them spawn where your not looking - an accompanying explosion sound, then an incendium grenade remains (Or fake explosion mark) beneath the fire with some fake weldable damage at the foot of the fire - as if an incendium grenade had been thrown.

**Reply by Kulik242** [reply]
Psychosis affected player could sometimes see other players as husks and clowns. And they may see a fake ballast flora.

**Reply by Udrakan** [reply]
I like the idea of psychotic players damaging devices. It has potential for some fun interactions with the crew.

**Comment by Benzin73** [comment]
Beisdes suffering different effects inside and outside the sub, wich i think is allready coming in the next version, I think more (and scarier) hallucitations would be nice. 
- You open a door on your sub, and a mudraptor jumps you (an imaginary one of course).
- You leave the nav console view or a periscope, and the whole room is splattered with liters of blood & bodyparts on the floor.
- More sounds! you suddenly hear screaming, you start hearing distant Husk sounds getting lowder & lowder (closer), etc.
- You suddenly see a clown with a knife running towards you.
- The chat box starts displaying messages of crew members you previously lost (if any) blaming you for their death.
- On periscope you see a huge sub firing railguns at you.

**Comment by Veritius** [comment]
Inaccurate or outright false sonar readings while psychotic. If the Captain's insane, they shouldn't be able to pilot the sub normally.

**Comment by Sviat** [comment]
Our best brainstorming on Psychosis effects was if Doctor is mad he starts seeing non-existent symptoms (including husk) in his goggles...

That makes so much fun >_<

**Comment by Working-Joe** [comment]
I'd like to add to all this that while I'd love to see these changes, it should be taken into account that newer players that do not know about Psychosis may run into trouble. Changes should also be made to make players realise they are under psychosis (at least after some time), so that they do not just think the game is bugged. Some ideas:
- Have bots suggest the player has psychosis if commands/reports are given for things that are not real multiple times
- Have a clear indicator for the player that psychosis is *over*, so that they know the previous events were because of psychosis.

This last part could be done in an interesting way by making Psychosis progressive. That is, have the effects of psychosis start as realistic, and then add more and more unrealistic events. This progression could end in the player passing out. This would also give a reason for medics to cure psychosis instead of just letting it run its course. 

It could even be made so that the player can take "fake" damage from the hallucinations, so that they believe they are actually dying from it - but once the fake damage becomes enough to kill them, instead of dying they pass out and then wake up without psychosis. 

It may also be fun to have the progressions relate to a certain theme. This team could be at random or with an increased chance relating to your job. Just a few ideas, listed as events that can occur after eachother:

**Water progression**
- The player occasionally hears running water 
- The player sees small leaks. Welding them actually stops the leaks. 
- The player starts to see larger leaks. Welding them stops the leaks. (leaks may be preceeded by a creature bonk)
- Crush depth music starts playing, there outside lights flicker and many leaks appear. Welding them does not stop the leaks
- The submarine floods appears to flood with water entirely.
- The submarine is gone. There is only the player in an endless abyss of water. The player passes out once they drown.

Psychosis ends if the player drowns to the fake water or the progression ends. 

**Wiring progression**
- The player occasionally hears sparks or buzzing.
- Occasionally a junction box appears broken. Repairing it is possible, but it takes quite a bit longer than usual. (pain.)
- The player will experience fake black-outs.
- The rods in the reactor appear (nearly) empty.
- The player starts seeing fires near junction boxes. It can be extinguished.
- After a fire, readings on JBs will show a too high load and the reactor will show a spike.
- The player will hear the nuclear reactor meltdown warning once they are close to the reactor. If the player runs away they will run slower and slower until they can't move if too far away. Inspecting the reactor shows a way too high temperature. They sliders can be moved but this will sway the output all over the place, until the reactor eventually explodes. 

The progression ends when dying to a fake fire or after the reactor explosion.

**Creature progression**
- The player occasionally hears creature noises outside
- The player sees blips on the sonar of animals that do not exist
- The player sees creatures through periscopes that do not exist. The creatures do die to coilgun damage. The creatures can also cause visible damage to the sub which can be repaired, similar to in the water progression.
- The player starts seeing other players as creatures like crawlers, mudraptors or husks when walking, treshers when swimming, or cells when near thalamus. Getting close to them causes the creatures to attack, dealing fake damage.

The progression ends when the player dies to a fake creature or some time after the creatures disappear.

**Health progression**
- Health items appear as other health items to the player, with increasingly bad mixups. From blood packs looking like bandages near the start up to morbusine looking like bandages.)
- The player will see increasingly worse afflictions on other players and themselves. From lacerations and burns near the start up to husk infection near the end. The afflictions will not affect the players, except the fake health of the player that has psychosis. They can be healed, however, with the item that *looks like* the correct cure. (it may actually be something completely different)

The progression ends when the player dies to (fake) medication or ailments.

**Clown progression**
- The player occasionally hears a honking horn or shoe squeaking.
- The player finds clown gear. This is only visible to them. They can pick it up and wear it. 
- The player can occasionally see a clown standing at the far end of rooms. Getting closer makes the clown fade out and disappear. If the player honks a horn, the clowns honk their horn back and disappear. 
- The player starts hearing faint clown music indefinitely.
- The player can see the honkmother in all her glory. They become unable to move. The honkmother starts walking towards the player. The clown music becomes louder and louder, then stops as the honkmother stands besides the player. The honkmother honks a horn. It is incredibly loud. The player passes out. 

The progression ends when the progression ends. This progression should take the longest of all, and can only be stopped by medication. If the player has worn the clown suit and has honked to all clowns, they will wake up with a random piece of the honkmother outfit upon waking up. 

Note: This is all just a dream. Although fun, implementing this would require rewriting many, many aspects of the game code and therefore this is not really achievable. (:

**Reply by Hotmud9** [reply]
### _**T H E H O N K M O T H E R**_

**Comment by RadCatish** [comment]
> "This last part could be done in an interesting way by making Psychosis progressive. That is, have the effects of psychosis start as realistic, and then add more and more unrealistic events. This progression could end in the player passing out. This would also give a reason for medics to cure psychosis instead of just letting it run its course."

Completely agree, this is a great idea

**Comment by Cogs16** [comment]
I think it would be awsome to utilize the pathfinding AI in psychosis. For instance, every couple of seconds your character might start moving towards the airlock unless you stop them. Maybe this should be a watcher effect only.
